Discovery/config_dialog.ui

```diff
@@ -246,17 +249,21 @@
    <item row="12" column="1">
     <widget class="QLineEdit" name="editBboxExpr"/>
    </item>
-   <item row="14" column="0" colspan="2">
-    <widget class="QDialogButtonBox" name="buttonBox">
-     <property name="orientation">
-      <enum>Qt::Horizontal</enum>
+   <item row="13" column="1">
+    <widget class="QCheckBox" name="timeCB">
+     <property name="text">
+      <string>Disable display time</string>
      </property>
-     <property name="standardButtons">
-      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
+    </widget>
+   </item>
+   <item row="15" column="0">
+    <widget class="QLabel" name="label_9">
+     <property name="text">
+      <string>Display time</string>
```

Comment:
I would suggest different wording to make it easier for users to understand - something like "[ ] Auto-hide map marker after [ 5 s ]"

Rather than using two lines, it would be better to keep this configuration in one line only.

Also, I would suggest to use a spin box widget rather than text edit widget - easier for the user to update the time. Spin box widget also supports specification of suffix - if you add suffix " s" then the value would be shown as e.g. "3 s"